docs-src/clusters/quick-install.md

```diff
@@ -22,7 +22,7 @@ It supports persistence to disk and in-memory mode through SQLite.
 
 **Prerequisites**
 
-Temporalite requires Go 1.18 or later.
+Temporalite requires [Go 1.18](https://go.dev/dl/) or later.
```

Comment:
```suggestion
Temporalite requires [Go 1.18 or later](https://go.dev/dl/).
```
Because Go is already up to 1.19.5, I recommend including "or later" in the link text to set expectations appropriately (and to simplify maintenance for us).